index.ts

```diff
@@ -11,9 +8,10 @@ import {
 } from "@safe-global/safe-modules-deployments";
 import dotenv from "dotenv";
 import { ethers } from "ethers";
+import { NearEthAdapter, MultichainContract } from "near-ca";
 
 dotenv.config();
-const { NEAR_EVM_ADDRESS, SAFE_SALT_NONCE, ERC4337_BUNDLER_URL } = process.env;
+const { SAFE_SALT_NONCE, ERC4337_BUNDLER_URL, ENTRY_POINT_V7 } = process.env;
```

Comment:
I don't think this should be an environment variable. The address is fixed because of `CREATE2` deployment, and the 4337 module won't allow anything else.